tests/unit/mock/config/parser/base/rad_etx/etx_full_received.py

```diff
@@ -0,0 +1,1262 @@
+from netutils.config.parser import ConfigLine
+
+data = [
+    ConfigLine(config_line='echo "Terminal Configuration"', parents=()),
```

Comment:
Should exit be returned?

Reply:
my thought was no, that keeps the same pattern that we did in forinet_fortios which has a very similar structure, with exit/end statements.  We removed them in that case. Happy to reconsider it but not sure what the exit would provide.